test/alphabet/alphabet_test.cpp

```diff
@@ -44,8 +44,7 @@ class alphabet : public ::testing::Test
 {};
 
 // add all alphabets here
-using alphabet_types = ::testing::Types<dna4, dna5, rna4, rna5, nucl16,
-                                        /*aa27,*/
+using alphabet_types = ::testing::Types<dna4, dna5, rna4, rna5, nucl16, aa27,
```

Comment:
not really important, but I had originally planned on each submodule's alphabets to be in one line :nail_care: